src/Symfony/Component/HttpKernel/Controller/ArgumentResolver.php

```diff
@@ -135,4 +141,21 @@ private function getPrettyName($controller): string
 
         return $controller;
     }
+
+    /**
+     * @param \Throwable[] $valueResolverExceptions
+     */
+    private function constructFailedToResolveException($controllerName, string $argumentName, array $valueResolverExceptions): \RuntimeException
```

Comment:
I'd suggest removing the extra method, that will remove one frame from stack traces and we don't need that split.